### PR TITLE
Hw23

### PR DESCRIPTION
**Pull request by Engelko**

### CODE REVIEW

File: Bash_scripts/HW23/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Ansible. Do for ubuntu and centos hosts.
+
+###  1.	Difference between ansible module: command, shell and raw
+###  2.	Create a role hw23. Inside the role:
+    
+      a.	Create file /opt/hw23_test_file. Set 644 permissions
+    
+      b.	Do in 2 ways. Rename the file /opt/hw23_test_file to hw23_test_file_exists if file exists(/opt/hw23_test_file) otherwise print debug message that FILE_NAME does not exist.
```
Comment (mavrik90) [inline Bash_scripts/HW23/README.md:8]: Did you do it in two ways?

File: Bash_scripts/HW23/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Ansible. Do for ubuntu and centos hosts.
+
+###  1.	Difference between ansible module: command, shell and raw
```
Comment (mavrik90) [inline Bash_scripts/HW23/README.md:3]: What is the difference?

Comment (Engelko) [reply Bash_scripts/HW23/README.md:3]: Command works with python, if python is not installed on the host, then the command will not work.
Raw is a module that works at a low level of Linux, without using python scripts, which allows pre-configuration on hosts that do not have python installed.
Shell is the most convenient of all, it allows you to work with pipline, operands

File: Bash_scripts/HW23/README.md (hunk 1 above)
Comment (Engelko) [reply Bash_scripts/HW23/README.md:8]: you can use Command as I did, and I suppose that you can still use the  module copy

**Review by mavrik90 — APPROVED** [review]
Approved!

**Pull request merged**